backend/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo 'Current PHP version: ' . phpversion() . "\n";
```

Comment:
Please add our https://github.com/blumilksoftware/codestyle to this project. You can do this now or in a separate issue/PR. It will help you avoid thing like using single and double quotes in one line. ;)

Reply:
Btw, this is better approach:
```suggestion
echo 'Current PHP version: ' . phpversion() . PHP_EOL;
```